pom.xml

```diff
@@ -12,7 +12,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.3.3.RELEASE</version>
+    <version>1.2.7.RELEASE</version>
```

Comment:
Just curious why we are downgrading.

Reply:
spring boot 2.x.x requires that we provide our own embedded database configuration for storing the state of the job execution, etc. 

I felt like it was taking me a bit too long to configure it properly at the time I was working on this so I figured I'd get back to doing that later since we'll need to do the same for the genome nexus annotation pipeline and cmo-pipelines projects when we upgrade the spring boot version. 

What do you think? Just deal with it now or is it fine to leave at this time?